ruby/lib/ci/queue/static.rb

```diff
@@ -63,7 +63,7 @@ def created_at=(timestamp)
       end
 
       def expired?
-        (@created_at.to_f TEN_MINUTES) < CI::Queue.time_now.to_f
+        (@created_at.to_f + TEN_MINUTES) < CI::Queue.time_now.to_f
```

Comment:
To be honest, we could also just return `false` here, static queue should never be expired.